SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/AlwaysInterestedTest/AlwaysInterestedTest.cpp

```diff
@@ -0,0 +1,220 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "AlwaysInterestedTest.h"
+#include "SpatialFunctionalTestFlowController.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/TestActors/AlwaysInterestedTestActors.h"
+
+#include "LoadBalancing/LayeredLBStrategy.h"
+
+#include "Net/UnrealNetwork.h"
+
+/**
+ * This test tests that AlwaysInterested UProperties are replicated to clients and servers correctly.
```

Comment:
I feel this test needs to also check that interest is correctly updated when the actor reference changes.

Reply:
Yeah good point - I'll add some additional testing.

Reply:
Tests added